### PR TITLE
Add self.requires_grad member for set dropout

### PR DESCRIPTION
**Pull request by Johnson-yue**
Possibly useful for finetuning the network

### CODE REVIEW

**Comment by MINGUKKANG** [comment]
Thank you Johnson-yue  :)


I am so happy that you are interested in my work.

Why do you think finetune is needed for the inception network?

I think the inception network is needed for just evaluation for GANs.

Sincerely, Minguk Kang

**Comment by Johnson-yue** [comment]
Because my dataset is very different from ImageNet , like glyph images. So I will finetune inception model for consistence compare result, and it is meaningful.
